src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/CourtCaseEventsService.kt

```diff
@@ -27,25 +27,38 @@ class CourtCaseEventsService(
   @Transactional(isolation = Isolation.SERIALIZABLE)
   fun processPersonFromCourtCaseEvent(person: Person) {
     log.debug("Entered processPersonFromCourtCaseEvent")
-    val pnc: String? = person.otherIdentifiers?.pncNumber
-    if (pncIdValidator.isValid(pnc)) {
-      val defendants = personRepository.findByDefendantsPncNumber(pnc!!)?.defendants.orEmpty()
+    if (person.otherIdentifiers?.pncNumber?.isNotEmpty() == true) {
+      val pncId = person.otherIdentifiers.pncNumber
```

Comment:
wouldn't we want to get the canonical form here?
presumably we are going to have to empty the database when this is merged?